src/post_processing.jl

```diff
@@ -623,10 +621,11 @@ function write_results(
     Q_load_vect = fill(0.0, N_BUS)
     bus_types = view(data.bus_type, :, time_step)
 
+    busIxToFAPower = _calculate_fixed_admittance_powers(sys, data)
     for (ix, bt) in enumerate(bus_types)
         P_load_vect[ix] = data.bus_activepower_withdrawals[ix, time_step] * sys_basepower
         Q_load_vect[ix] = data.bus_reactivepower_withdrawals[ix, time_step] * sys_basepower
-        P_admittance, Q_admittance = _get_fixed_admittance_power(sys, buses[ix], data, ix)
+        (P_admittance, Q_admittance) = get(busIxToFAPower, ix, (0, 0))
```

Comment:
nitpick: type stability
```suggestion
        (P_admittance, Q_admittance) = get(busIxToFAPower, ix, (0.0, 0.0))
```